src/utils/store.js

```diff
@@ -1,8 +1,124 @@
-const store = require('store2');
-const overflow = require('./store.overflow');
+const session = {
+  overflow: {},
+  get(key) {
+    // Overflow
+    const valueFromOverflow = this.overflow[key];
 
-// Inject overflow into store
-// https://raw.githubusercontent.com/nbubna/store/master/src/store.overflow.js
-overflow(store, store._);
+    if (valueFromOverflow) {
+      return valueFromOverflow;
+    }
+
+    const valueFromSession = sessionStorage.getItem(key);
+
+    if (valueFromSession) {
+      return JSON.parse(valueFromSession);
+    }
+
+    return null;
+  },
+  set(key, value) {
+    try {
+      return sessionStorage.setItem(key, JSON.stringify(value));
+    } catch (e) {
+      if (e.name === 'QUOTA_EXCEEDED_ERR'
+        || e.name === 'NS_ERROR_DOM_QUOTA_REACHED'
+        || e.toString().indexOf('QUOTA_EXCEEDED_ERR') !== -1
+        || e.toString().indexOf('QuotaExceededError') !== -1) {
+        this.overflow[key] = value;
+        return null;
+      }
+
+      throw e;
+    }
+  },
+  remove(key) {
+    if (this.overflow?.[key]) {
+      delete this.overflow[key];
+    }
+
+    return sessionStorage.removeItem(key);
+  },
+  key(i) {
+    const keyFromSessionStorage = sessionStorage.key(i);
+    const sessionStorageLength = sessionStorage.length;
+
+    if (i >= sessionStorageLength) {
+      const overflowIndex = i - sessionStorageLength;
+
+      return Object.keys(this.overflow)?.[overflowIndex];
+    }
+
+    return keyFromSessionStorage;
+  },
+  length() { return sessionStorage.length + Object.keys(this.overflow || {}).length; },
```

Comment:
I believe `this.overflow` will always be an object and it's defaulted to empty object, so the `|| {}` is probably not necessary.